src/main/java/frc/robot/subsystems/RobotPoses.java

```diff
@@ -58,24 +61,27 @@ public void periodic() {
     // exist. Instead, we'll log where we *want* the mechanisms to be and assume
     // they get there instantly.
     if (Robot.isSimulation()) {
-      elevatorPos = subElevator.getLastDesiredPosition().in(Units.Meters) / 2;
-      algaeAngle = subAlgaeIntake.getLastDesiredPivotAngle().in(Units.Degrees);
+      elevatorPos = subElevator.getLastDesiredPosition().div(2);
+      algaeAngle = subAlgaeIntake.getLastDesiredPivotAngle();
     } else {
```

Comment:
Looks good just one bug to fix
```suggestion
      algaeAngle = subAlgaeIntake.getLastDesiredPivotAngle().unaryMinus();
```